static/templates/admin_user_group_list.hbs

```diff
@@ -13,7 +13,7 @@
             {{t 'Discard changes' }}
         </button>
         <button class="button rounded small delete btn-danger">
-            {{t 'Delete' }}
+            <i class="fa fa-trash-o" aria-hidden="true"></i>
```

Comment:
This should use `aria-label` and `title` (both as "delete" with translation tags)  -- otherwise this UI widget is not accessible to screen readers.

Reply:
<i class="fa fa-trash-o" aria-label="delete" title="delete"></i>
is good or,
<i class="fa fa-trash-o" aria-hidden="true"></i>

In templates\admin_emoji_list.hbs the code is like:
<i class="fa fa-trash-o" aria-hidden="true"></i>